lib/seahorn/BvOpSem2RawMemMgr.hh

```diff
@@ -206,30 +280,29 @@ public:
   /// \param[in] mem is the memory value being read from
   /// \param[in] ty is the type of value being loaded
   /// \param[in] align is the known alignment of the load
-  Expr loadValueFromMem(PtrTy ptr, MemValTy mem, const llvm::Type &ty,
-                        uint64_t align) override;
+  Expr loadValueFromMem(const PtrTy &ptr, const MemValTy &mem,
```

Comment:
note for future. `loadValueFromMem` and `storeValueToMem` can be implemented in the mixin because they are implemented completely in terms of other functions of the memory manager.

Reply:
added note to mixin class